.gitignore

```diff
@@ -14,7 +14,6 @@ node_modules/
 
 ### Node ###
 # Logs
-logs
```

Comment:
**Note** This is an unrelated change. I noticed some changes to the `Logs.jsx` file was ignored and found out that this was the reason.